app/controllers/api/auth/stories_controller.rb

```diff
@@ -32,7 +38,12 @@ def filtered(resources)
   end
 
   def resources_base
-    @stories ||= current_user.approved_account_stories
+    # If there is a network_id specified, use that network
```

Comment:
see line 43, where if there is a network id, then the network visible stories are a subset of `super.published`